lib/byron/exe/cardano-wallet-byron.hs

```diff
@@ -220,18 +245,68 @@ cmdServe = command "serve" $ info (helper <*> helper' <*> cmd) $ mempty
                                  Options
 -------------------------------------------------------------------------------}
 
+data NetworkConfiguration where
+    MainnetConfig
+        :: (SomeNetworkDiscriminant, NodeVersionData)
+        -> NetworkConfiguration
+
+    TestnetConfig
+        :: (SomeNetworkDiscriminant, NodeVersionData)
+        -> FilePath
+        -> NetworkConfiguration
+
+-- | Hand-written as there's no Show instance for 'NodeVersionData'
```

Comment:
I wonder if why Show and not ToText here?

Reply:
That's a good question. I will dig a bit more about why `Show` is needed for `MsgNeworkName`